nomad/structs/structs.go

```diff
@@ -6922,6 +6922,11 @@ func validateServices(t *Task, tgNetworks Networks) error {
 			}
 		}
 
+		// GH-9450: connect block is only allowed on group level
```

Comment:
```suggestion
		// connect block is only allowed on group level
```

We don't include the GitHub issue ID in our comments (except in the rare case where we've leaving a known-bug in place). Source control gives us all the context we need.